WooCommerce/Classes/System/DefaultFeatureFlagService.swift

```diff
@@ -8,7 +8,7 @@ struct DefaultFeatureFlagService: FeatureFlagService {
         case .addProductVariations:
             return buildConfig == .localDeveloper || buildConfig == .alpha
         case .shippingLabelsRelease1:
-            return buildConfig == .localDeveloper || buildConfig == .alpha
+            return true
```

Comment:
I'm just curious. The reason for setting the flag to true without removing it is so that it's easier to roll back the feature, if needed?

Reply:
yup exactly! I created another issue https://github.com/woocommerce/woocommerce-ios/issues/3434 as the last issue of the project to remove the feature flag when the feature has been beta tested, launched, and the analytics looking similar to Android 🙂